lib/min_heap.rb

```diff
@@ -14,18 +14,30 @@ def initialize
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n) where n is the number of items in the heap
+  # Space Complexity: O(1)
```

Comment:
Since your `heap_up` method is recursive you have to consider the call stack in terms of the space complexity.